packages/webview_flutter/example/pubspec.yaml

```diff
@@ -1,29 +1,23 @@
 name: webview_flutter_tizen_example
 description: Demonstrates how to use the webview_flutter_tizen plugin.
 
-# The following line prevents the package from being accidentally published to
-# pub.dev using `pub publish`. This is preferred for private packages.
-publish_to: 'none' # Remove this line if you wish to publish to pub.dev
-
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.2.2 <3.0.0"
```

Comment:
Is there any special reason for this? ``2.7.0`` -> ``2.2.2 ``

Reply:
There is no reason. I referenced just other plugin's version.